kubernetes/README.md

```
@@ -0,0 +1,245 @@
+---
+
+## **7. Verify Deployment**
+
+### **Check Deployed Resources**
+```bash
+kubectl get all -n bankapp-namespace
+```
+
+### **Access the Application**
+- Open your browser and navigate to:
+  ```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)